test/cqlpy/cassandra_tests/validation/operations/select_order_by_test.py

```diff
@@ -317,8 +317,7 @@ def testOrderByForInClause(cql, test_keyspace):
             assert_invalid_message(cql, table, "LIMIT must be strictly positive",
                                  "SELECT v as c2 FROM %s where pk1 = ? AND pk2 IN (?, ?) ORDER BY c1 DESC , c2 DESC LIMIT 0; ", 1, 1, 2)
 
-# Reproduces #7751
-@pytest.mark.xfail(reason="Issue #7751")
+@pytest.mark.skip(reason="Issue #22061")
```

Comment:
Why did you need to skip this test instead of just "xfail" as usual? Does it crash Scylla?
If it doesn't crash Scylla, xfail is preferred (you can notice xpass, and it's easier to run with pytest or test/cqlpy/run with "--runxfail" option).

Reply:
I didn't use xfail because it crashes

Reply:
This is so sad :-( It doesn't crash, it has an on_internal_error, but somebody decided our **tests** need to crash in this case, whereas in a normal Scylla server an on_internal_error is just a normal exception (see https://github.com/scylladb/scylladb/issues/13352 for more).
So now I understand.

Reply:
Tests need to crash so we have a nice coredump to debug.

Reply:
I don't agree, but we won't solve this difference today :-) An on-internal-error is a  clear error in a clear place, and it's usually (and in this case) reproducible so this core dump is not helpful and even counterproductive and in this case - forces us to skip a test.